webapp/src/components/VideoFeed.tsx

```diff
@@ -31,25 +35,26 @@ export function VideoFeed() {
     };
 
     const feedUrl = `http://${videoHost}/video_feed?rand=${rand}`;
-
+    const isHidden = !isActive || isLoading || errorMsg;
+    let imgStyle = {};
+    let src = feedUrl;
+    let alt = "video feed";
+    if (isHidden) {
+        imgStyle = { display: "none" };
+        src = "/please-stand-by.png";
+        alt = "please stand by";
+    }
```

Comment:
The above is not quite right.  if you have imgStyle `{display: "none"}` then you'll never see the "please stand by"  which is desirable for locding and error states.